lib/pkgcloud/openstack/orchestration/client/resources.js

```diff
@@ -62,13 +62,25 @@ exports.getResource = function (stack, resource, callback) {
  * @param {function}          callback      f(err, resources) where stacks is an array of Resource
  * @returns {*}
  */
-exports.getResources = function (stack, callback) {
+exports.getResources = function (stack, options, callback) {
   var self = this;
 
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+
   function getResources(stack) {
-    return self._request({
+    var requestOptions = {
       path: urlJoin('/stacks', stack.name, stack.id, 'resources')
-    }, function (err, body) {
+    };
+
+    requestOptions.qs = [];
```

Comment:
Why not move this initialization into the definition of `requestOptions` on line 74?